### ISSUE TITLE
Video Rotation

### ISSUE THREAD

**Issue by guodan** [RenderHeads/UnityPlugin-AVProVideo#1639]
### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE

**Describe the issue**
After I update the AVPro Video from 2.1.8 to 2.8.5, the video rotation is 90°. 

**Your Setup (please complete the following information):**
 - Unity version: 2020.3.46f1
 - AVPro Video version (number and edition (trial/core/ultra/enterprise)): core 2.8.5
 - Operating system version: win10
 - Device model: iPad mini 5th / iPad Pro 12.9 5th
 - Video specs (resolution, frame-rate, codec, file size): 

**To Reproduce**
1. Import package [UnityNativeCamera](https://github.com/yasirkula/UnityNativeCamera)
2. Use UnityNativeCamera record a video. 
3. Use AVPro Video play the video with local path from step2.

**Videos**
This is play with 2.1.8
https://github.com/RenderHeads/UnityPlugin-AVProVideo/assets/18362550/264309c5-bdfd-4e1b-a39b-3767956b5554

This is play with 2.8.5
https://github.com/RenderHeads/UnityPlugin-AVProVideo/assets/18362550/234a567a-ff8d-4f0e-891c-2a9c54460b25

### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE
Instead email the link to us unitysupport@renderheads.com

**Comment by Chris-RH** [comment]
This sounds like what was mentioned at the end of https://github.com/RenderHeads/UnityPlugin-AVProVideo/issues/1627

Are you able to provide one of the videos that it is trying to play? unitysupport@renderheads.com

**Comment by Chris-RH** [comment]
How are you playing the video?
Rendertexture?
Apply to mesh? material?
Display UG/IMGUI?

**Comment by guodan** [comment]
Display UGUI

**Comment by MorrisRH** [comment]
There is an error in how the texture transform is being created for videos that are rotated 180 degrees (landscape flipped). To fix this you need to change line 316 of `AVProVideo/Runtime/Scripts/Internal/Helper.js` from:
```csharp
private static Matrix4x4 LandscapeFlippedMatrix = Matrix4x4.TRS(new Vector3(0f, 1f, 0f), Quaternion.Euler(0f, 0f, -90f), Vector3.one);
```
to:
```csharp
private static Matrix4x4 LandscapeFlippedMatrix = Matrix4x4.TRS(new Vector3(1f, 1f, 0f), Quaternion.Euler(0f, 0f, -180f), Vector3.one);
```
We'll make sure this fix is included in the next release.

**Comment by Chris-RH** [comment]
This has been fixed in AVPro Video version 2.9.0. Please let us know if is still not working for you.

**Comment by 1a1aca** [comment]
> 此问题已在 AVPro Video 版本 2.9.0 中修复。如果仍然不适合您，请告诉我们。
It has this problem again after version 3.0, which makes me unable to continue using it. Please fix it, thank you.
